CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKIconsRenderable.java

```diff
@@ -1023,6 +994,8 @@ private int UpdateVertexUniformBuffers(MemoryStack stack) {
             if (VkFailed(ret)) { return ret; }
         }
         
+        UpdatePushConstantsMatrix();
```

Comment:
I think this should be UpdateVertexPushConstants() to be consistent with the UniformBuffer names.  Also naming the contents doesn't tell us what matrix or help us if we change the matrix to a vec4 and ivec2 in the future.

Reply:
I can rename, thought push constants can be used over multiple shader stages, which is why I used the push constant type for this and the hit test

Reply:
Changed

Reply:
True, but so can UBs (though we don't).